network.py

```diff
@@ -75,7 +76,43 @@ def from_byte_S(self, byte_S):
         dst = byte_S[0 : NetworkPacket.dst_S_length].strip('0')
         data_S = byte_S[NetworkPacket.dst_S_length : ]        
         return self(dst, data_S)
-    
+
+class MPLSFrame:
+    ## packet encoding lengths
+    label_length = 20
+    dst_S_length = 5
+
+    ##@param dst: address of the destination host
+    # @param data_S: packet payload
+    # @param priority: packet priority
+    def __init__(self, label, dst, data_S, priority=0):
+        self.dst = dst
+        self.data_S = data_S
+        if label is None:
+            self.label = str(random.randint(1, 10000000)).zfill(self.label_length)
+        else:
+            self.label = label
+        #TODO: add priority to the packet class
```

Comment:
Is this a TODO for this part of the assignment, or was this something necessary later that you were reminding yourself about?

Reply:
That's the comment he has in there for part 3